app/livechat/client/views/app/tabbar/chatRoomHistoryItem.html

```diff
@@ -0,0 +1,26 @@
+<template name='chatRoomHistoryItem'>
+	<li class='list-chat' id='{{_id}}' aria='{{v.token}}' >
+		<div class="history-div">
+			<div class="visitor-img">
+				<img src="/avatar/{{responseBy.username}}" alt="">
+			</div>
+			<div class="visitor-history-details">
+				<div class="visitor-details">
+					<h1>{{responseBy.username}}</h1>
+						{{#if open}}
+							<p class='open'>Open</p>
+						{{else}}
+							<p>Close at time {{time}}</p>
+						{{/if}}
+				</div>
+				<div class="total-messages">
+					<p>{{room.msgs}} messages</p>
+				</div>
+				<div class="agent-comment">
```

Comment:
The closing room message is not mandatory, there is a setting that forces the agent to leave a message when closing the conversation. So, this block of code is conditional, the template will need to provide a helper method that will be used to check if the closing message exists or not.
Something like:

- `hasClosingRoomMessage`

Reply:
done